donate/thunderbird/templates/payment/thank_you.html

```diff
@@ -3,7 +3,7 @@
 {% load i18n util_tags %}
 
 {% block thank_you_description %}
-    <p>{% blocktrans %}Your donation will go right to work protecting the web, keeping it open and available to all.{% endblocktrans %} {% blocktrans with email='give@thunderbird.net' %}We’ve emailed you a donation receipt; if it’s missing, please check your junk/spam folders, then contact us at <a href="mailto:{{ email }}">{{ email }}</a>.{% endblocktrans %}</p>
+    <p>{% blocktrans %}Your donation will go right to work protecting the web, keeping it open and available to all.{% endblocktrans %} {% blocktrans with email='thunderbird-donate@mozilla.org' %}We’ve emailed you a donation receipt; if it’s missing, please check your junk/spam folders, then contact us at <a href="mailto:{{ email }}">{{ email }}</a>.{% endblocktrans %}</p>
```

Comment:
This is email change is temporary... wondering if updating the `email` var here is okay or this will affect localization?

Reply:
This won’t affect localization. In fact, emails and URLs were moved to vars so that we could update them without impacting existing translations (and also so that they can’t be edited by mistake during the translation process). That’s paying off today :)

Reply:
woohoo